client.go

```diff
@@ -118,3 +124,30 @@ func newClientUsingAddresses(port uint16, remoteAddresses []string) (*Client, er
 		return newClientUsingCluster(cluster)
 	}
 }
+
+// Get is a simple invocation of the NewFetchValueCommand
+func (c *Client) Get(opts *GetOptions) ([]*Object, error) {
```

Comment:
Good points on both.

The only reason I initially broke the "Options" out was so for Set, you can have some flexibility for how the client can set any 2i properties, but maybe if they need that, they need something above a simple Set?

Either way, happy to make these changes